### PR TITLE
[ideep] Fuse Conv-Relu after IDEEP graph rewrite, skip group conv

### PR DESCRIPTION
**Pull request by vishar0**
IDEEP supports fusion for non-group conv

### CODE REVIEW

**Review by yinghai — APPROVED** [review]
Cool!

File: caffe2/opt/optimize_ideep.cc [hunk 1]
```diff
@@ -21,6 +22,13 @@ void OptimizeForIdeep(repr::NNModule* nn) {
       return false;
     }
 
+    // IDEEP doesn't support fusion group conv
+    int group =
+        ArgumentHelper::GetSingleArgument<OperatorDef, int>(*op, "group", 1);
```
Comment (bddppq) [inline caffe2/opt/optimize_ideep.cc]: There is `HasArgument`

Comment (yinghai) [reply caffe2/opt/optimize_ideep.cc]: We still need to check whether it's 1 or not. If it's 1, it's ok. So GetSingleArgument here looks ok.

**Comment by yinghai** [comment]
Test failure is due to CI infra error.

**Pull request merged**